packages/peregrine/lib/talons/Gallery/useGalleryItem.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+import { useLazyQuery } from '@apollo/react-hooks';
+
+/**
+ *
+ * @param {Object} props
+ * @param {Object} props.item - the product item being rendered
+ * @param {Object} props.queries - the queries for the gallery item
+ */
+export const useGalleryItem = props => {
```

Comment:
This talon is rare in that it doesn't return anything (returns `undefined`). Consider documenting that in the JSDoc comments.

Reply:
eac357918